src/Bicep.Core/LanguageConstants.cs

```diff
@@ -73,6 +73,38 @@ public static class LanguageConstants
         public const string ResourceDependsOnPropertyName = "dependsOn";
         public const string TypeNameString = "string";
 
+        public static readonly string[] StandardizedResourcePropertyNames = new[]
```

Comment:
>StandardizedResourcePropertyNames [](http://example.com/codeflow?start=40&length=33)

I'm wondering if we should call these two properties:
* `ReadWriteDeployTimeConstantPropertyNames`
* `WriteOnlyDeployTimeConstantPropertyNames`

Regardless of the above, could you also add a comment to this one as well?

Reply:
Yeah, those are better names. Added a comment.